draft-pda-protocol.md

```diff
@@ -437,6 +434,33 @@ Time uint64; /* seconds elapsed since start of UNIX epoch */
 * `proto`: The PPM protocol, e.g., Prio or Hits. The rest of the structure
   contains the protocol specific parameters.
 
+An `Aggregator` is structured as follows:
+
+~~~
+struct {
+  AggregatorId id;
+  Url endpoint;
+} Aggregator;
+
+uint8 AggregatorId;
```

Comment:
I would consider making this larger. I could imagine there being > 256 aggregators.

Reply:
Would it actually make sense to not have an integer in this field and only in the submissions, and just have those values refer to indexes here.

Reply:
I agree, there's no reason not to make this a 64 bit integer (or even a much longer byte string as I suggest in another thread on this PR). However I'm not sure if we want the IDs in other messages to refer to indexes here, because that imposes the new constraint on aggregator IDs that they be sequential, which is not required with the current `Aggregator` definition.